gunicorn/arbiter.py

```diff
@@ -495,6 +495,17 @@ def manage_workers(self):
             (pid, _) = workers.pop(0)
             self.kill_worker(pid, signal.SIGTERM)
 
+        if self.pidfile is None and self.cfg.pidfile is not None and self.cfg.delay_pidfile:
+            worker_values = list(self.WORKERS.values())
+            booted = True
+            for worker in worker_values:
+                if not worker.booted:
```

Comment:
It should handle this case fine. The initial run of manage_workers is not likely to succeed in creating a pidfile. A later run should succeed.